ci-operator/step-registry/ipi/deprovision/aws/blackholenetwork/ipi-deprovision-aws-blackholenetwork-commands.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+
+if test ! -f "${SHARED_DIR}/blackholenetworkregion"
+then
+  echo "No blackholenetworkregion, so unknown AWS region, so unable to tear down."
+  exit 0
+fi
+
+REGION="$(cat "${SHARED_DIR}/blackholenetworkregion")"
+STACK_NAME="$(cat "${SHARED_DIR}/blackholenetworkstackname")"
+
+# cleaning up after ourselves
+aws --region "${REGION}" cloudformation delete-stack --stack-name "${STACK_NAME}" &
+wait "$!"
```

Comment:
Not that I know of in this particular instance. I was trying to keep similar style as other steps using cloudformation

Reply:
I'm suspecting this could have to do with the amount of output generated. I've added a stack delete complete output, but still running in the background like https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/deprovision/proxy/ipi-deprovision-proxy-commands.sh

Reply:
Ah, makes sense.